extensions/hacker-news-top-stories/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "hacker-news-top-stories",
+  "title": "Hacker News Top Stories",
+  "description": "A menubar extension to view top stories on Hacker News based on a configurable points threshold.",
+  "icon": "icon.png",
+  "author": "KevinBatdorf",
+  "contributors": [
+    "pernielsentikaer"
+  ],
+  "categories": [
+    "News"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "view-top-stories",
+      "title": "Hacker News Top Stories Menu Bar",
+      "description": "View Hacker News stories based on a configurable points threshold which are submitted over the last 24 hours.",
```

Comment:
I guess this isn't true. It's more like if they reach the threashhold they will show up for 24 hours. So one from 5 days ago might hit 500 points today, then it will show up for 24 hours in the menu bar